server/controllers/finance/accounts/references.js

```diff
@@ -288,13 +287,30 @@ async function lookupAccountReference(id) {
   return reference;
 }
 
+/**
+ * @function getAccountsForReferenceHTTP
+ *
+ * @description
+ * HTTP interface to compute the accounts associated with a particular reference.
+ */
+async function getAccountsForReferenceHTTP(req, res, next) {
```

Comment:
Why did you add `HTTP` in the name of the function, is that a convention?

Reply:
There is another function name almost the same thing here.  So, to make sure it wasn't confusing, I named this function that takes in `req`, `res`, `next` "HTTP".  It isn't really a convention - just lack of creativity on my part.

If you can think of a  better name, I would be open to it :laugh: